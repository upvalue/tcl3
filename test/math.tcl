puts [== 0 0]
puts [>= 5 4]
puts [<= 4 5]
puts [>= 5 5]
puts [<= 5 5]
puts [< 4 5]
puts [> 5 4]
puts [!= 1 2]

puts [+ 2 2]
puts [- 6 2]
puts [/ 16 4]
puts [* 2 2]
