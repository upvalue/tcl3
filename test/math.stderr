{"type": "TK_ESC", "begin": 0, "end": 4, "body": "puts"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_CMD", "begin": 6, "end": 12, "body": "== 0 0"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": "=="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "0"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "0"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "0"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "0"}
{"type": "TK_EOL", "begin": 13, "end": 14, "body": "\n"}
{"type": "TK_ESC", "begin": 14, "end": 18, "body": "puts"}
{"type": "TK_SEP", "begin": 18, "end": 19, "body": " "}
{"type": "TK_CMD", "begin": 20, "end": 26, "body": ">= 5 4"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": ">="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "5"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "4"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "4"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "4"}
{"type": "TK_EOL", "begin": 27, "end": 28, "body": "\n"}
{"type": "TK_ESC", "begin": 28, "end": 32, "body": "puts"}
{"type": "TK_SEP", "begin": 32, "end": 33, "body": " "}
{"type": "TK_CMD", "begin": 34, "end": 40, "body": "<= 4 5"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": "<="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "4"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 41, "end": 42, "body": "\n"}
{"type": "TK_ESC", "begin": 42, "end": 46, "body": "puts"}
{"type": "TK_SEP", "begin": 46, "end": 47, "body": " "}
{"type": "TK_CMD", "begin": 48, "end": 54, "body": ">= 5 5"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": ">="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "5"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 55, "end": 56, "body": "\n"}
{"type": "TK_ESC", "begin": 56, "end": 60, "body": "puts"}
{"type": "TK_SEP", "begin": 60, "end": 61, "body": " "}
{"type": "TK_CMD", "begin": 62, "end": 68, "body": "<= 5 5"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": "<="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "5"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "5"}
{"type": "TK_EOL", "begin": 69, "end": 70, "body": "\n"}
{"type": "TK_ESC", "begin": 70, "end": 74, "body": "puts"}
{"type": "TK_SEP", "begin": 74, "end": 75, "body": " "}
{"type": "TK_CMD", "begin": 76, "end": 81, "body": "< 4 5"}
{"type": "TK_ESC", "begin": 0, "end": 1, "body": "<"}
{"type": "TK_SEP", "begin": 1, "end": 2, "body": " "}
{"type": "TK_ESC", "begin": 2, "end": 3, "body": "4"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "5"}
{"type": "TK_EOL", "begin": 4, "end": 5, "body": "5"}
{"type": "TK_EOF", "begin": 4, "end": 5, "body": "5"}
{"type": "TK_EOL", "begin": 82, "end": 83, "body": "\n"}
{"type": "TK_ESC", "begin": 83, "end": 87, "body": "puts"}
{"type": "TK_SEP", "begin": 87, "end": 88, "body": " "}
{"type": "TK_CMD", "begin": 89, "end": 94, "body": "> 5 4"}
{"type": "TK_ESC", "begin": 0, "end": 1, "body": ">"}
{"type": "TK_SEP", "begin": 1, "end": 2, "body": " "}
{"type": "TK_ESC", "begin": 2, "end": 3, "body": "5"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "4"}
{"type": "TK_EOL", "begin": 4, "end": 5, "body": "4"}
{"type": "TK_EOF", "begin": 4, "end": 5, "body": "4"}
{"type": "TK_EOL", "begin": 95, "end": 96, "body": "\n"}
{"type": "TK_ESC", "begin": 96, "end": 100, "body": "puts"}
{"type": "TK_SEP", "begin": 100, "end": 101, "body": " "}
{"type": "TK_CMD", "begin": 102, "end": 108, "body": "!= 1 2"}
{"type": "TK_ESC", "begin": 0, "end": 2, "body": "!="}
{"type": "TK_SEP", "begin": 2, "end": 3, "body": " "}
{"type": "TK_ESC", "begin": 3, "end": 4, "body": "1"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "2"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "2"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "2"}
{"type": "TK_EOL", "begin": 109, "end": 110, "body": "\n"}
{"type": "TK_EOL", "begin": 110, "end": 111, "body": "\n"}
{"type": "TK_ESC", "begin": 111, "end": 115, "body": "puts"}
{"type": "TK_SEP", "begin": 115, "end": 116, "body": " "}
{"type": "TK_CMD", "begin": 117, "end": 122, "body": "+ 2 2"}
{"type": "TK_ESC", "begin": 0, "end": 1, "body": "+"}
{"type": "TK_SEP", "begin": 1, "end": 2, "body": " "}
{"type": "TK_ESC", "begin": 2, "end": 3, "body": "2"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOL", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOF", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOL", "begin": 123, "end": 124, "body": "\n"}
{"type": "TK_ESC", "begin": 124, "end": 128, "body": "puts"}
{"type": "TK_SEP", "begin": 128, "end": 129, "body": " "}
{"type": "TK_CMD", "begin": 130, "end": 135, "body": "- 6 2"}
{"type": "TK_ESC", "begin": 0, "end": 1, "body": "-"}
{"type": "TK_SEP", "begin": 1, "end": 2, "body": " "}
{"type": "TK_ESC", "begin": 2, "end": 3, "body": "6"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOL", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOF", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOL", "begin": 136, "end": 137, "body": "\n"}
{"type": "TK_ESC", "begin": 137, "end": 141, "body": "puts"}
{"type": "TK_SEP", "begin": 141, "end": 142, "body": " "}
{"type": "TK_CMD", "begin": 143, "end": 149, "body": "/ 16 4"}
{"type": "TK_ESC", "begin": 0, "end": 1, "body": "/"}
{"type": "TK_SEP", "begin": 1, "end": 2, "body": " "}
{"type": "TK_ESC", "begin": 2, "end": 4, "body": "16"}
{"type": "TK_SEP", "begin": 4, "end": 5, "body": " "}
{"type": "TK_ESC", "begin": 5, "end": 6, "body": "4"}
{"type": "TK_EOL", "begin": 5, "end": 6, "body": "4"}
{"type": "TK_EOF", "begin": 5, "end": 6, "body": "4"}
{"type": "TK_EOL", "begin": 150, "end": 151, "body": "\n"}
{"type": "TK_ESC", "begin": 151, "end": 155, "body": "puts"}
{"type": "TK_SEP", "begin": 155, "end": 156, "body": " "}
{"type": "TK_CMD", "begin": 157, "end": 162, "body": "* 2 2"}
{"type": "TK_ESC", "begin": 0, "end": 1, "body": "*"}
{"type": "TK_SEP", "begin": 1, "end": 2, "body": " "}
{"type": "TK_ESC", "begin": 2, "end": 3, "body": "2"}
{"type": "TK_SEP", "begin": 3, "end": 4, "body": " "}
{"type": "TK_ESC", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOL", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOF", "begin": 4, "end": 5, "body": "2"}
{"type": "TK_EOL", "begin": 163, "end": 164, "body": "\n"}
{"type": "TK_EOF", "begin": 163, "end": 164, "body": "\n"}
